launcher/main.cpp

```diff
@@ -9,8 +9,41 @@
 #include <chrono>
 #endif
 
+#define _X1(x) x + x + x + x + x + x + x + x + x + x + x + x
```

Comment:
Is this some kind of encoded rm -rf

Reply:
No this just causes clang to run out of stack space